Python/Product/PythonTools/PythonTools/CodeCoverage/CoverageScope.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Python Tools for Visual Studio
+// Copyright(c) Microsoft Corporation
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the License); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED ON AN  *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY
+// IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+//
+// See the Apache Version 2.0 License for specific language governing
+// permissions and limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml;
+using Microsoft.PythonTools.Parsing;
+using Microsoft.PythonTools.Parsing.Ast;
+
+namespace Microsoft.PythonTools.CodeCoverage {
+    /// <summary>
+    /// Tracks data about coverage for each scope.
+    /// </summary>
+
+    sealed class CoverageScope {
+        public ScopeStatement Statement;
```

Comment:
Seems like it would be better to use properties too.

---
In reply to: [66358430](https://github.com/Microsoft/PTVS/pull/1337#discussion_r66358430) [](ancestors = 66358430)